activemodel/lib/active_model/errors.rb

```diff
@@ -147,9 +147,9 @@ def set(key, value)
 
     # Delete messages for +key+. Returns the deleted messages.
     #
-    #   person.errors[:name]    # => ["cannot be nil"]
+    #   person.errors[:name]        # => ["cannot be nil"]
```

Comment:
because as far as I know, this is a convention. All output messages needs to be aligned.